components/infobox/wikis/dota2/infobox_company_custom.lua

```diff
@@ -0,0 +1,47 @@
+---
+-- @Liquipedia
+-- wiki=dota2
+-- page=Module:Infobox/Company/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local Company = Lua.import('Module:Infobox/Company', {requireDevIfEnabled = true})
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomCompany = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomCompany.run(frame)
+	local company = Company(frame)
+	_args = company.args
+
+	company.createWidgetInjector = CustomCompany.createWidgetInjector
```

Comment:
Out of interest, could this be shortened like this:
```suggestion
	company.createWidgetInjector = CustomInjector
```
As a call `comparny.createWidgetInjector()` would be `CustomInjector()` then?

Reply:
Should like something we can do
__call metatable on the Injector

Reply:
Yeah should be fine doing, thought should be changed everywhere if we decide to do this (and probably either rename the field (eg `createWidgetInjector` -> `widgetInjector`) or change it into a function to be with the Injector as a param (`company.setWidgetInjector(CustomInjector)`)

Resolving as out of scope for this PR